src/rules.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2020-2021 the Deno authors. All rights reserved. MIT license.
 use crate::context::Context;
-use dprint_swc_ecma_ast_view::Program as ProgramView;
+
+pub use ast_view::Program;
+pub use ast_view::ProgramRef;
```

Comment:
How about moving these `pub use` to the crate root (i.e. src/lib.rs)?
These two types are now included in the public interface of this crate, and also it'd be more intuitive for them to live and re-exported in the crate root than to be here as they don't seem to be related to "rules" very much.

Reply:
Done in c9e2996e0c362130deb79cfc994d9041dde560bb.